server/viame_server/viame.py

```diff
@@ -39,6 +39,7 @@ def __init__(self):
         self.resourceName = "viame"
         self.static_pipelines = None
 
+        self.route("GET", ("dataset",), self.list_datasets)
```

Comment:
Torn on if this should stay or not.  Web version doesn't call this endpoint ever even though it is placed in the API for the desktop version.  It's nice to have parity if we decide we need this information ever on the web version.  If we do keep it I think we may need a slightly more descriptive name indicating that it is 'a list of all datasets that this user has access to'.

Reply:
I vote remove it.